sale_commission/views/account_invoice_view.xml

```diff
@@ -22,7 +35,9 @@
                 <field name="company_id" position="after">
                     <field name="commission_free"/>
                     <field name="agents"
-                           attrs="{'readonly': [('commission_free', '=', True)]}"/>
+                           attrs="{'readonly': [('commission_free', '=', True)]}"
+                           widget="one2many_tags"
+                    />
```

Comment:
Should we use the one2many_tags here?
Maybe leave this view with the default widget, since it didn't cause any problems here.. and it provides more information.